modules/ixBidAdapter.js

```diff
@@ -678,14 +753,14 @@ function trimImpressions(impressions, maxSize) {
  */
 
 function removeFromSizes(bannerSizeList, bannerSize) {
-  for (let i = 0; i < bannerSizeList.length; i++) {
-    if (bannerSize[0] == bannerSizeList[i][0] && bannerSize[1] == bannerSizeList[i][1]) {
-      bannerSizeList.splice(i, 1);
-      return true;
+  if (bannerSize && bannerSize.length > 0) {
+    for (let i = 0; i < bannerSizeList.length; i++) {
+      if (bannerSize[0] == bannerSizeList[i][0] && bannerSize[1] == bannerSizeList[i][1]) {
```

Comment:
findindex requires a polyfill

Reply:
Right, forgot it wasn't in IE11. Thanks for flagging this.
In that case, this could still be simplified with a for-of and made more self-consistent with the rest of the file by using ===.

```
for (let size of bannerSizeList) {
  if (bannerSize[0] === size[0] && bannerSize[1] === size[1]) {
    bannerSizeList.splice(i, 1);
    break;
  }
}
```

Reply:
This wont be able to splice due to missing index.

Reply:
Right, missed that in my update. Just the === in that case to be consistent